Gemfile

```diff
@@ -37,3 +38,6 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 
 
+
+
+gem "faker", "~> 2.18", :groups => [:development, :test]
```

Comment:
You already have a `:development, :test` group declared on line 19. So you can move the gem faker to there. :nail_care: